src/components/Camera/StandardCamera.js

```diff
@@ -224,8 +251,43 @@ const StandardCamera = ( ): Node => {
         </View>
       </View>
       <Snackbar visible={showAlert} onDismiss={() => setShowAlert( false )}>
-        {t( "You-can-only-upload-20-media" )}
+        {t( "You-can-only-upload-20-media " )}
       </Snackbar>
+      <StandardBottomSheet snapPoints={["20%"]} hide={discardState.hide}>
+        <View className="px-[20px]">
+          <View className="relative flex items-center justify-center mt-[22px]">
+            <Heading3>{t( "discard-photos" )}</Heading3>
+            <View className="absolute right-0">
+              <IconButton
+                icon="close-button-x"
+                onPress={() => setDiscardState( INITIAL_DISCARD_STATE )}
+                accessibilityLabel={t( "Cancel" )}
+                accessibilityState={{ disabled: false }}
+              />
+            </View>
+          </View>
+          <Body3 className="my-[20px] ml-[10px]">
+            {t( "discard-photos-description" )}
```

Comment:
I believe we're trying to name all i18n strings with the actual content of the paragraph, rather than a generic name. So a better key name would be `By-existing-your-photos-will-not-be-saved`.

Also, this is centered and uses `List2` in the latest Figma designs.